### PR TITLE
chore: supabase v2 docs

### PR DESCRIPTION
**Pull request by khaya-zulu**
Update supabase guide.

### CODE REVIEW

**Comment by peterp** [comment]
@khaya-zulu could you ask one of the other team members to review this document? Perhaps Andrew or Jude?

File: docs/06-tutorials/supabase-clone-environments.md [hunk 1]
```diff
@@ -1,115 +1,97 @@
...
-
-![Snaplet onboarding connect your database](../../static/screenshots/supabase-integration/onboarding_connect_db.png)
+1. **A production Supabase project’s connection string:** This can be found in Supabase via Organization > All Projects > Your Project > Project Settings > Connection String. We’ll refer to this in the guide as your “source database” - where your data comes from.
+2. **A development Supabase project’s connection string:** Same steps as above, but a different project/environment. We’ll refer to this in the guide as your “target database” - where we’re restoring the snapshot of the source database to. If you haven’t got this set up yet, we’ll take you through the process of setting this up.
 
-You’ll then want to enter the credentials of your production Supabase project. Find the "Connection string" in Supabase via `Organization > Project > Settings > Database > Connection string` (At the bottom of the page.)
+> **Highly Recommended:** Create a read-only role for your connection string. For more info on how to create a read-only role across all schemas, you can check out [our docs](/guides/postgresql#create-a-read-only-role).
```
Comment (avallete) [inline docs/06-tutorials/supabase-clone-environments.md:28]: **suggestion**

Maybe could be good to precise that this is true for the "source database" aka, the one who access production data. For the target, we need to have write access to it in order to restore the snapshot.

File: docs/06-tutorials/supabase-clone-environments.md [hunk 2]
```diff
@@ -1,115 +1,97 @@
...
+- Transforming (skipping will leave all your table columns untouched).
 
-At this point, Snaplet will automatically detect any columns that have Personally Identifiable Information (PII) and mark them in purple. If there are any additional columns that hold data you would like to anonymise, you can click on the respective column name and provide a replacement value for the data in that column. Once you are happy and satisfied, you can click on `Review and Save` and proceed to the dashboard.
+Once you have gone through those steps, a new snapshot process will start and you will now be on the “capture step.” Wait for it to finish - this is the snapshot you will restore into your **target database.**
 
-![Snaplet onboarding create snapshot](../../static/screenshots/supabase-integration/onboarding_create_snapshot.png)
+<img src="/screenshots/onboarding_capture.png" alt="Snaplet onboarding capturing your database" style={{ border: "1px solid #e5e7eb" }} />
 
-### Step 3: Create a Snapshot
+### Step 3: Create a target database on Supabase
```
Comment (avallete) [inline docs/06-tutorials/supabase-clone-environments.md]: **suggestion**

Or maybe the precision about the need for the target to be able to write could go here.

**Review by avallete — APPROVED** [review]
Looks good to me.

File: docs/06-tutorials/supabase-clone-environments.md (hunk 2 above)
Comment (khaya-zulu) [reply docs/06-tutorials/supabase-clone-environments.md]: You're right, I updated the headings.

**Review by avallete — APPROVED** [review]

**Comment by khaya-zulu** [comment]
Thanks :)

**Pull request merged**